README.rst

```diff
@@ -31,7 +31,8 @@ Alternatively, clone and install with:
 
 Documentation
 -------------
-`PyAnsys <https://grantami.docs.pyansys.com>`_
```

Comment:
This seems like a long hyperlink, but it matches the style and I can't see a more sensible place to start the link.